### DISCUSSION TITLE
Credit not visible for https://github.com/advisories/GHSA-4g6q-77j7-vvjc

### DISCUSSION THREAD

**Discussion by abhishekwebcode** [community/community#125644] [Code Security]
### Select Topic Area

Bug

### Body

CVE: CVE-2023-6460
Github Link: https://github.com/advisories/GHSA-4g6q-77j7-vvjc
OFFICIAL CVE LINK: https://www.cve.org/CVERecord?id=CVE-2023-6460
My name is not visible under the credits section of GITHUB ADVISORY but its visible under cve.org link above.
Thus the 1 security advisory credit is also not showing on my profile
For example: You can see credits in right bottom here https://github.com/advisories/GHSA-f8m6-h2c7-8h9x

**Comment by jacopotediosi** [comment]
I see you in the credits section

**Comment by abhishekwebcode** [comment]
hey @jacopotediosi , when i created this thread, i also raised a support ticket since i didnt know how to get it through ( https://support.github.com/ticket/personal/0/3086170 ) github resolved it promptly

Some suggestions:
I had to request google vrp to add my name, therefore you can see github didnt automatically pick my name from finder role under credits section of this CVE. Maybe instead of only seeing the CVE during creation also check for people added during CVE update, the references section can have github links to PR etc which will have the github username.
Github updated my security credit after i requested support to look into it.

Appreciate your looking into this matter though.
